tests/tamr_client/datasets/test_record.py

```diff
@@ -154,6 +155,20 @@ def test_delete_infer_primary_key():
     assert snoop["payload"] == utils.stringify(deletes)
 
 
+@responses.activate
+def test_stream():
+    s = utils.session()
+    dataset = utils.dataset()
+
+    url = tc.URL(path="datasets/1/records")
+    responses.add(
+        responses.GET, str(url), body="\n".join([json.dumps(x) for x in _records_json])
```

Comment:
You should use a generator expression instead of list comprehension to avoid unnecessary `[]` and to avoid loading the data into memory separately from creating the `str`:

```suggestion
        responses.GET, str(url), body="\n".join(json.dumps(x) for x in _records_json)
```